package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "dnd_beyond_overlay",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.1",
+    "@testing-library/user-event": "^7.2.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.0",
+    "react-dom": "^16.13.0",
+    "react-scripts": "3.4.0"
```

Comment:
This isn't too much of a concern because this is an app, not a library, but generally dependencies are organized into either `dependencies` or `devDependencies`:
* `dependencies` is for dependencies that your app needs at runtime (in your case: `prop-types`, `react`, `react-dom`)
* `devDependencies` is for basically everything else, dependencies that you need for development processes (in your case: `@testing-library/*`, `react-scripts`, `eslint*`)

`dependencies` are added with `npm install --save {dep}`, and `devDependencies` are added with `npm install --save-dev {dep}`.

If this was a library, npm would make sure to only install transitive dependencies via the `dependencies` list and ignore everything under `devDependencies`, so you'd definitely want to follow this guideline to reduce the amount of data that dependent packages need to download. However, organizing like this can also be useful for apps because running `npm install --only=prod` will ignore `devDependencies`, which is generally the procedure you'd follow for production deployments to reduce your bundle size.

If you'd like to fix this, you can simply move the lines from the `dependencies` list to the `devDependencies` list. The lock file doesn't distinguish the two types of dependencies.

Reply:
That's strange because I'm pretty sure `react-create-app` generated the dependencies file like this. In a webextension environment, I don't think the bundle size matters as much, but it's definitely something i'll keep an eye on